2001-02-16  Jakub Jelinek  <jakub@redhat.com>

	* sysdeps/unix/sysv/linux/alpha/syscalls.list (ftruncate): Add
	__ftruncate64 alias.
	* sysdeps/unix/sysv/linux/sparc/sparc64/syscalls.list (ftruncate):
	Likewise.

--- libc/sysdeps/unix/sysv/linux/alpha/syscalls.list.jj	Fri Aug 25 19:17:40 2000
+++ libc/sysdeps/unix/sysv/linux/alpha/syscalls.list	Fri Feb 16 11:57:11 2001
@@ -26,7 +26,7 @@ fstatfs		-	fstatfs		2	__fstatfs	fstatfs 
 statfs		-	statfs		2	__statfs	statfs statfs64
 getrlimit	-	getrlimit	2	__getrlimit	getrlimit getrlimit64
 setrlimit	-	setrlimit	2	__setrlimit	setrlimit64 setrlimit
-ftruncate	-	ftruncate	2	__ftruncate	ftruncate ftruncate64
+ftruncate	-	ftruncate	2	__ftruncate	ftruncate __ftruncate64 ftruncate64
 truncate	-	truncate	2	truncate	truncate64
 
 # these are actually common with the x86:
--- libc/sysdeps/unix/sysv/linux/sparc/sparc64/syscalls.list.jj	Fri Aug 25 18:59:54 2000
+++ libc/sysdeps/unix/sysv/linux/sparc/sparc64/syscalls.list	Fri Feb 16 11:57:52 2001
@@ -8,7 +8,7 @@ fstatfs		-	fstatfs		2	__fstatfs	fstatfs 
 statfs		-	statfs		2	__statfs	statfs statfs64
 getrlimit	-	getrlimit	2	__getrlimit	getrlimit getrlimit64
 setrlimit	-	setrlimit	2	__setrlimit	setrlimit64 setrlimit
-ftruncate	-	ftruncate	2	__ftruncate	ftruncate ftruncate64
+ftruncate	-	ftruncate	2	__ftruncate	ftruncate __ftruncate64 ftruncate64
 truncate	-	truncate	2	truncate	truncate64
 mmap		-	mmap		6	__mmap		mmap __mmap64 mmap64
 
